lib/beaker/options/parser.rb

```diff
@@ -139,12 +142,9 @@ def set_default_host!(hosts)
           end
         end
 
-        if not default.empty?
-          #we already have a default set, do nothing
-          if default.length > 1
-            parser_error "Only one host may have the role 'default', default roles assigned to #{default}"
-          end
-        else
+        # default_set? will throw an error if length > 1
```

Comment:
Might be outside the scope of this PR, but if we're interested in cleanup in here, this block might be a candidate for a method extraction. This `set_default_host!` method is getting a bit unwieldy.